terraform/node_module_expand.go

```diff
@@ -5,15 +5,19 @@ import (
 
 	"github.com/hashicorp/terraform/addrs"
 	"github.com/hashicorp/terraform/configs"
+	"github.com/hashicorp/terraform/lang"
+	"github.com/hashicorp/terraform/states"
 )
 
 // nodeExpandModule represents a module call in the configuration that
 // might expand into multiple module instances depending on how it is
 // configured.
 type nodeExpandModule struct {
 	CallerAddr addrs.ModuleInstance
```

Comment:
Note that these fields are not final yet, as there won't actually be a `ModuleInstance` at this point.